modules/macos_directory_cleaner/files/clean_before_reboot.sh

```diff
@@ -10,4 +10,7 @@ if command -v directory_cleaner &> /dev/null
 then
     directory_cleaner -c /opt/directory_cleaner/configs/config.toml --remove-empty-directories /opt/worker/downloads
     directory_cleaner -c /opt/directory_cleaner/configs/config.toml --remove-empty-directories /Users/cltbld/Library/Caches/Mozilla/updates/opt/worker/tasks/
+    directory_cleaner -c /opt/directory_cleaner/configs/config.toml --remove-empty-directories /Users/cltbld/Downloads
+    directory_cleaner -c /opt/directory_cleaner/configs/config.toml --remove-empty-directories /Users/cltbld/Desktop
+    directory_cleaner -c /opt/directory_cleaner/configs/config.toml --remove-empty-directories /Users/cltbld/Library/Application\ Support/Firefox/Crash\ Reports
```

Comment:
There is also the `Library/Caches/Firefox` folder which contains profiles and possible downloaded updates. Maybe this should be cleaned up as well?